tests/compat.rs

```diff
@@ -1,44 +1,32 @@
-use super::*;
+use rpm::*;
 use std::io::prelude::*;
 use std::io::BufReader;
 use std::process::Stdio;
-use std::str::FromStr;
 
-fn test_rpm_file_path() -> std::path::PathBuf {
-    let mut rpm_path = cargo_manifest_dir();
-    rpm_path.push("test_assets/389-ds-base-devel-1.3.8.4-15.el7.x86_64.rpm");
-    rpm_path
-}
-
-fn cargo_manifest_dir() -> std::path::PathBuf {
-    std::path::PathBuf::from(env!("CARGO_MANIFEST_DIR"))
-}
-
-fn cargo_out_dir() -> std::path::PathBuf {
-    cargo_manifest_dir().join("target")
-}
+mod common;
 
-#[cfg(feature = "signature-meta")]
 use signature::{self, Verifying};
 
-#[cfg(feature = "signature-pgp")]
 mod pgp {
```

Comment:
Why are these removed?

Reply:
If you look in Cargo.toml, the entire test files are flagged on those features, so putting the flags in the code isn't necessary